luigi/notifications.py

```diff
@@ -254,19 +251,15 @@ def send_email_sendgrid(sender, subject, message, recipients, image_png):
     client.send(to_send)
 
 
-def _email_disabled():
+def _email_disabled_reason():
     if email().format == 'none':
-        logger.info("Not sending email when email format is none")
-        return True
+        return "Not sending email when email format is 'none'"
```

Comment:
Good point. @daveFNbuck, was `none` preserved from your previous refactoring?

Although, maybe we still want to keep this so people can pass cli parameter `--email-format-none`. But it's a bit unlikely, so I'm still okay with removing this.